FranDepascuali/RubyTrainingApp
# Adding spanish language

## Sumary

Text can now be shown in English or Spanish.

**Trello cards**:
https://trello.com/c/5nKCedVQ/11-internacionalizar-el-blog
## Test Plan
- Open "application.rb" and config.i18n.default_locale = :es for spanish or :en for english 
- Open a terminal.
- Go to project directory.
- Run ./bin/rails server.
- Open browser and go to http://localhost:3000/.
- Text should be in spanish.
- Server must be reloaded to change language.
## Screenshots

Spanish:
![spanish](https://cloud.githubusercontent.com/assets/12101394/8455980/c2057e5c-1fde-11e5-889d-27b862826ae6.gif)

English:
![english](https://cloud.githubusercontent.com/assets/12101394/8456015/08626ff4-1fdf-11e5-8679-343968438862.gif)
## Known issues
- When throwing errors, variable is shown as its in the model. ie: Body is shown as "body", not as "comentario" (spanish language)
